### PR TITLE
Ascendex modifyMarginHelper, addMargin, reduceMargin

### PR DESCRIPTION
**Pull request by Dan-krm**
Added methods that allow you to increase or decrease the margin on perpetual swaps for ascendex:
https://ascendex.github.io/ascendex-futures-pro-api-v2/#change-margin-for-isolated-positions
```
ascendex.addMargin (BTC/USDT:USDT, 1)
1462 ms
{
  info: { code: '0' },
  type: 'add',
  amount: '1',
  code: 'USDT',
  symbol: 'BTC/USDT:USDT',
  status: 'ok'
}
```
```
ascendex.reduceMargin (BTC/USDT:USDT, -1)
1463 ms
{
  info: { code: '0' },
  type: 'reduce',
  amount: '-1',
  code: 'USDT',
  symbol: 'BTC/USDT:USDT',
  status: 'ok'
}
```

### CODE REVIEW

File: js/ascendex.js [hunk 1]
```diff
@@ -2135,6 +2137,52 @@ module.exports = class ascendex extends Exchange {
...
+            'type': type,
+            'amount': amount,
+            'code': market['quote'],
+            'symbol': market['symbol'],
+            'status': status,
+        };
+    }
+
+    async reduceMargin (symbol, amount, params = {}) {
+        if (amount >= 1) {
```
Comment (frosty00) [inline js/ascendex.js:2173]: The argument to reduceMargin is actually a positive number, let's remove these checks

File: js/ascendex.js [hunk 2]
```diff
@@ -2135,6 +2137,52 @@ module.exports = class ascendex extends Exchange {
...
+            throw new BadRequest (this.id + ' reduceMargin() amount argument must be a negative number');
+        }
+        return await this.modifyMarginHelper (symbol, amount, 'reduce', params);
+    }
+
+    async addMargin (symbol, amount, params = {}) {
+        if (amount < 1) {
```
Comment (frosty00) [inline js/ascendex.js:2180]: also this one, the amount could be less than one

File: js/ascendex.js (hunk 1 above)
Comment (Dan-krm) [reply js/ascendex.js:2173]: With the endpoint for ascendex to reduce margin you need to enter a negative number. Should I remove these checks then convert the number to a negative in modifyMarginHelper if reduceMargin is called?

Comment (kroitor) [reply js/ascendex.js:2173]: @Dan-krm i'd say let's just accept any number (positive or negative) in both reduceMargin and addMargin and then pass that to modifyMarginHelper as is.

**Pull request merged**